src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -123,44 +146,44 @@ public Task<Result<PropertyDTO[]>> GetPropertiesAsync(ConnectionDTO connection,
                 : AppendQueryString(QualityProfileListAPI, "?projectKey={0}", request.ProjectKey);
 
             return InvokeSonarQubeApi(connection, query, token,
-                stringResponse => JObject.Parse(stringResponse)["profiles"].ToObject<QualityProfileDTO[]>());
+                stringResponse => JObject.Parse(stringResponse)["profiles"].ToObject<QualityProfileResponse[]>());
         }
 
-        public Task<Result<RoslynExportProfile>> GetRoslynExportProfileAsync(ConnectionDTO connection,
+        public Task<Result<RoslynExportProfileResponse>> GetRoslynExportProfileAsync(ConnectionRequest connection,
             RoslynExportProfileRequest request, CancellationToken token)
         {
             const string QualityProfileExportAPI = "api/qualityprofiles/export"; // Since 5.2
 
             var roslynExporterName = string.Format(CultureInfo.InvariantCulture, "roslyn-{0}",
-                        request.Language.Key);
+                        request.LanguageKey);
             var query = AppendQueryString(QualityProfileExportAPI, "?name={0}&language={1}&exporterKey={2}",
-                request.QualityProfileName, request.Language.Key, roslynExporterName); // TODO: should handle optional params
+                request.QualityProfileName, request.LanguageKey, roslynExporterName); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)